Makefile

```diff
@@ -80,9 +80,12 @@ verify: toolchain tidy download ## Verify code. Includes dependencies, linting,
 	cp $(KARPENTER_CORE_DIR)/pkg/apis/crds/* pkg/apis/crds
 	yq -i '(.spec.versions[0].additionalPrinterColumns[] | select (.name=="Zone")) .jsonPath=".metadata.labels.karpenter\.azure\.com/zone"' \
 		pkg/apis/crds/karpenter.sh_nodeclaims.yaml
+	hack/validation/kubelet.sh
 	hack/validation/labels.sh
 	hack/validation/requirements.sh
 	hack/validation/common.sh
+	cp pkg/apis/crds/* charts/karpenter-crd/templates
```

Comment:
Curious why this changed?

Wasn't the mirroring of info already handled?

Reply:
The change likely has to do with webhook configuration injection, see hack/mutation/conversion_webhooks_injection.sh